draksetup - Check root permissions instead of crashing

Example (no xl in path, we're not root)
```
$ draksetup test
[2021-07-16 07:21:05,157][INFO] Checking xen-detect...
Running in PV context on Xen V4.15.
[2021-07-16 07:21:05,171][INFO] Testing if xl tool is sane...
[2021-07-16 07:21:05,179][ERROR] Failed to test xl info command. There might be some dependency problem (please execute 'xl info' manually to find out).
Traceback (most recent call last):
  File "/opt/venvs/drakrun/lib/python3.7/site-packages/drakrun/draksetup.py", line 218, in sanity_check
    check=True,
  File "/usr/lib/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command 'xl info' returned non-zero exit status 127.
```

--- a/drakrun/drakrun/draksetup.py
+++ b/drakrun/drakrun/draksetup.py
@@ -205,6 +205,9 @@ def cleanup():
 
 
 def sanity_check():
+    if not check_root():
+        return False
+
     logging.info("Checking xen-detect...")
     proc = subprocess.run("xen-detect -N", shell=True)
 

